packages/semantic-ui/src/ArrayFieldItemTemplate/ArrayFieldItemTemplate.tsx

```diff
@@ -1,45 +1,45 @@
 import React from "react";
+import {
+  ArrayFieldTemplateItemType,
+  GenericObjectType,
+  getUiOptions,
+} from "@rjsf/utils";
 import { Button, Grid, Segment } from "semantic-ui-react";
 
 import { MaybeWrap } from "../util";
 
-const gridStyle = (vertical) => ({
+const gridStyle = (vertical: boolean) => ({
   display: "grid",
   gridTemplateColumns: `1fr ${vertical ? 65 : 110}px`,
 });
 
-// checks if it's the first array item
```

Comment:
@jacqueswho do you remember why you used this odd logic to determine the first array item rather than simply using the `index` like I did below? This always seems to return true from what the tests indicate

Reply:
I don't remember doing this code, could have been a contribution